### PR TITLE
feat: #65 Custom timestamp parameter

### LINKED ISSUE #65: Support providing own timestamp
In order to be better integrated with logging system, such as `winston`, it will be nice if `raygun4node` allows user to provide their own timestamp (from `winston`) when reporting errors

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Adds functionality for custom timestamp when reporting errors, requested in #65
- **Approach**: Adds the `timestamp` parameter to the `send()` method as optional config param.
- Closes #65 

**Type of change**

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

**Updates**

- Add `timestamp` to `SendParameters`.
- `timestamp` uses the standard `Date` type or can be passed as `number`.
- Implemented tests
- Updated README docs

### Test plan :test_tube:

- Tested in example.
- Implement tests

### Author to check :eyeglasses:
- [x] Project and all contained modules builds successfully
- [x] Self-/dev-tested 
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Comment by miquelbeltran** [comment]
Note: No need to include in 1.1.0, wait until release to merge

File: README.md
Comment (sumitramanga) [inline README.md:219]: Is it only these 2 formats that can be used?

File: lib/raygun.ts
Comment (sumitramanga) [inline lib/raygun.ts:252]: Is this meant to have `_` ?

File: README.md
Comment (sumitramanga) [inline README.md:204]: It would be  good to add why this could be use or when this could be useful (e.g. the Issue mentioned with the usage of `winston`

File: README.md
Comment (miquelbeltran) [reply README.md:219]: Yes, I expect most users are either using `Date` or ms timestamps.

File: lib/raygun.ts
Comment (miquelbeltran) [reply lib/raygun.ts:252]: Yes, because the `timestamp` variable is already declared.

**Review by sumitramanga — APPROVED** [review]
Cheers for the clarity and docs change, looks good 🚀

**Pull request merged**
